xbmc/addons/AddonInstaller.cpp

```
@@ -828,6 +828,8 @@ bool CAddonUnInstallJob::DoWork()
   }
 
   ClearFavourites();
+  if (m_remove)
+    CFileUtils::DeleteItem("special://profile/addon_data/"+m_addon->ID()+"/", true);
```

Some use path substitution. Would this silently fail on that?